frontend/src/app/modules/intervention-form/file-uploader/file-uploader.component.html

```diff
@@ -1,27 +1,24 @@
-<section class="attachments">
-  <h2 class="mat-h2">Załączniki</h2>
-  <span *ngIf="files.length" class="file-counter">Wybrano {{ files.length }}</span>
+<span *ngIf="files.length" class="file-counter">Wybrano {{ files.length }}</span>
 
-  <div class="uploader-container">
-    <p *ngIf="!files.length && !editTemporaryDisabled" class="add-files-text">{{ addFilesMessage }}</p>
-    <p *ngIf="editTemporaryDisabled" class="add-files-text">Edytowanie załączników chwilowo niedostępne</p>
```

Comment:
Temporary edit disabled switch is not needed, as adding new files will not be blocked during edit.
The problem is with removing existing files - for that I will use a separate component in the future :)